taskcluster/mozillavpn_taskgraph/worker_types.py

```diff
@@ -3,7 +3,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from voluptuous import Any, Required, Optional
-
+from six import text_type
```

Comment:
I'm assuming you copy/pasted this from somewhere, but there's no need to support Py2 anymore, so let's `s/text_type/str` and not depend on `six` here :)